schema/transportation/segment.yaml

```diff
@@ -51,6 +57,71 @@ properties:
         default: []
 "$defs":
   propertyDefinitions:
+    destinationLabelType:
+      description: >-
+        The type of object of the destination label.
+      type: string
+      enum: 
+      - place
+      - street
+      - country 
+      - route
+    destinations:
+      description: >-
+        Information about what destinations can be reached if following this segment. 
+        Includes references to the segments from which it is needed to transition 
+        onto this segment to reach the destinations.
+      type: array
+      items:
+        type: object
+        unevaluatedProperties: false
+        required: [labels, from, to_connector_id, final_heading]
+        properties:
+          labels:
+            description: >-
+              Describes the destination labels on the directions sign.
+            type: array
+            items:
+              type: object
+              unevaluatedProperties: false
+              required: [value, type]
+              properties:
+                value:
+                  description: >-
+                    Names the object that is reached
+                  type: string
+                  pattern: ^(\S.*)?\S$
+                type: { "$ref": "#/$defs/propertyDefinitions/destinationLabelType" }
+            minItems: 1
+            uniqueItems: true
+          symbols:
+            type: array
+            items:
+              description: >-
+                A symbol or icon present on the sign.
+              type: string
+              pattern: ^(\S.*)?\S$
+          from:
+            description: >-
+              List of segments-connector pairs from where we need to transition 
+              to this segment to reach the destination.
+            type: array
+            items:
+              description: Pair of segment and connector IDs along the sequence
+              "$ref": "#/$defs/propertyDefinitions/subSegmentReference"
+              when:
+              allOf:
+                - "$ref": "#/$defs/propertyContainers/headingScopeContainer"
+          to_connector_id:
+            description: >-
+              Identifies the point of physical connection between the
+              previous segment in the sequence and the segment in this
+              sequence entry.
+            type: string                  
+          final_heading:
```

Comment:
`final_heading` seems weird here. It makes more sense on prohibited transitions because they are referring to a future sequence you need to take. But here, you're literally on the segment.

We should either rename it to `heading` or replace it with a mandatory `when: heading:` block. I think the second one is more consistent with the transportation model as a whole.